packages/republik-crowdfundings/graphql/resolvers/_mutations/submitPledge.js

```diff
@@ -295,33 +299,38 @@ module.exports = async (_, args, context) => {
           }
         } else if (!user) {
           // create user
-          user = await transaction.public.users.insertAndGet(
-            {
-              email: pledge.user.email,
-              firstName: pledge.user.firstName,
-              lastName: pledge.user.lastName,
-              birthday: pledge.user.birthday,
-              phoneNumber: pledge.user.phoneNumber,
-            },
-            { skipUndefined: true },
```

Comment:
this `skipUndefined` was unnecessary because `insertAndGet` doesn't need it right? Otherwise we would need one in the insert pledge below. But as far as I understand the pogi docs not necessary or even an real option on [`InsertOption`](https://pogi.readthedocs.io/API/QueryOptions/#sqlqueryoption-insertoption).

Reply:
Yes, `skipUndefined` is thus far just for querying (find, update, etc.) but useless while inserting.